src/workerd/io/compatibility-date.capnp

```diff
@@ -441,4 +441,13 @@ struct CompatibilityFlags @0x8f8c1b68151b6cef {
     $compatDisableFlag("fetch_legacy_url")
     $compatEnableDate("2024-06-03");
   # Ensures that WHATWG standard URL parsing is used in the fetch API implementation.
+
+  nodeJsCompatV2 @50 :Bool
+      $compatEnableFlag("nodejs_compat_v2")
+      $compatDisableFlag("no_nodejs_compat_v2")
+      $experimental;
+  # Implies nodeJSCompat with the following additional modifications:
+  # * Node.js Compat built-ins may be imported/required with or without the node: prefix
+  # * Node.js Compat common globals are available everywhere (e.g. Buffer, process)
+  # * node:process cannot be replaced by the worker bundle
```

Comment:
Where is this implemented?

Reply:
oh right... decided not to go with this. Just need to remove this bullet point